docs/admin/configure-cache-auth.md

```diff
@@ -0,0 +1,120 @@
+# Configuring Cache Server with authentication
+
+In addition to default Cache Server instance configured [here](configure-cache.md), you have option to enable authenticated cache. This chapter describes all the steps needed. 
+
+Packages installed: `stashcache-daemon fetch-crl stashcache-cache-server xrootd-lcmaps globus-proxy-utils`
+
+## Authenticated Cache server
+
+:heavy_exclamation_mark: Make sure you've in place following pre-requisities from [install step here](install.md):
+* __Service certificate:__ create copy of the certificate to `/etc/grid-security/xrd/xrd{cert,key}.pem`
+   * set owner of the directory `/etc/grid-security/xrd/` to `xrootd:xrootd` user:
+      ```
+      $ chown -R xrootd:xrootd /etc/grid-security/xrd/
+      ```
+* __Network ports__: allow connections on port `8443 (TCP)` 
+
+:heavy_exclamation_mark: Beware, authenticated cache requires presence of the [config file](../configs/xrootd-stashcache-cache-server.cfg) `/etc/xrootd/xrootd-stashcache-cache-server.cfg`. 
+
+Now, create symbolic link to existing configuration file with `-auth` postfix:
+```
+   [root@client ~]$ cd /etc/xrootd/
+   [root@client ~]$ ln -s xrootd-stashcache-cache-server.cfg xrootd-stashcache-cache-server-auth.cfg
+```
+
+### RHEL7
+
+On RHEL7 system, you need to configure and run following systemd units:
+* `xrootd@stashcache-cache-server-auth.service`
+* `xrootd-renew-proxy.service`
+* `xrootd-renew-proxy.timer`
+* `fetch-crl-cron`
+
+#### Auth.service
+1. Enable `xrootd@stashcache-cache-server-auth.service` instance:
+```
+   [root@client ~]$ systemctl enable xrootd@stashcache-cache-server-auth
+```
+
+2. Reload daemons:
+```
+   [root@client ~]$ systemctl daemon-reload
+```
+
+#### Proxy.service
+1. Create the file with following content:
+```
+   [root@client ~]$ cat /usr/lib/systemd/system/xrootd-renew-proxy.service
```

Comment:
I don't see the place where you create the `override.conf` file to specify the proxy to the auth-stashcache